site/_posts/2018-04-04-quandries.md

```diff
@@ -0,0 +1,33 @@
+---
+title: quandries
+author: d-m-u
+date: 2018-04-04
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Since its founding in 2018, Last Month In MIQ has been the absolute golden standard for journalistic integrity, never afraid to grab truth by the scruff of the neck and shake it up a little. Of course, that being said, we do ever so occasionally take a break from our regularly scheduled program to bring you insights into the minds of our crack team here at MIQ. I know you look forward to such rare occasions with baited breath. Best not to think about what it's baited with.  
```

Comment:
Isn't it `bated`? ok nvm... :stuck_out_tongue:

Reply:
This one isn't wrong!

Reply:
I reali~z~sed :wink: